plugins/chat/spec/requests/chat/api/channel_threads_controller_spec.rb

```diff
@@ -124,6 +124,15 @@
       )
     end
 
+    it "is ok for threads where the OM has mentions" do
```

Comment:
"is ok" is a little vague IMO. I think we can improve this test by ensuring that the mentions are displayed in the threaded message? Perhaps we can just fabricate an additional message with a mention for an existing test?

Reply:
Oh yeah true I just wrote it to confirm my assumption but then didn't update the text :laughing: I'll also improve it to make sure the mention is displayed -- it's just a problem if a thread original message has a mention